cmd/snap/cmd_info_test.go

```diff
@@ -64,9 +65,12 @@ type flushBuffer struct{ bytes.Buffer }
 func (*flushBuffer) Flush() error { return nil }
 
 func (s *infoSuite) TestMaybePrintServices(c *check.C) {
+	var buf flushBuffer
+	iw := snap.NewInfoWriter(&buf)
 	for _, infos := range [][]client.AppInfo{svcAppInfos, mixedAppInfos} {
-		var buf flushBuffer
-		snap.MaybePrintServices(&buf, &client.Snap{Name: "foo", Apps: infos})
+		buf.Reset()
+		snap.SetupDiskSnap(iw, "", &client.Snap{Name: "foo", Apps: infos})
```

Comment:
any particular reason to use SetupDiskSnap a bit more here and other places vs SetupSnap,  the latter is more the common case in a sense?

Reply:
no good reason. I preferred writing `""` over `nil, nil` ?

Reply:
I like all the `nil`s :)